### PR TITLE
fix: only show actionable txs on dashboard

### PR DESCRIPTION
**Pull request by iamacook**
## What it solves

This is a proof of concept based on a suggestion earlier in the week. Already signed transactions appearing on the dashboard,

## How this PR fixes it

Queued transactions that have already been signed by the currently connected wallet are filtered from the dashboard. If they are fully confirmed and ready to execute they are displayed.

## How to test it

Queue a transaction on a Safe with a >1 threshold. Observe that the transaction does not appear on the dashboard of the creator until the threshold is met by the other signers.

### CODE REVIEW

File: src/components/dashboard/PendingTxs/PendingTxsList.tsx [hunk 1]
```diff
@@ -48,21 +49,29 @@ const StyledEmptyCard = styled(Card)`
...
-  const queuedTxsToDisplay = uniqBy(queuedTxns, (item) =>
-    isMultisigExecutionInfo(item.transaction.executionInfo) ? item.transaction.executionInfo.nonce : '',
-  ).slice(0, size)
+  const queuedTxsToDisplay = uniqBy(queuedTxns, (item) => {
+    isMultisigExecutionInfo(item.transaction.executionInfo) ? item.transaction.executionInfo.nonce : ''
+  })
+    .filter(({ transaction }) => {
+      return isMultisigExecutionInfo(transaction.executionInfo) &&
+        transaction.txStatus === TransactionStatus.AWAITING_CONFIRMATIONS
+        ? transaction.executionInfo?.missingSigners?.some(({ value }) => value === wallet?.address)
```
Comment (katspaugh) [inline src/components/dashboard/PendingTxs/PendingTxsList.tsx:71]: `sameAddress(value, wallet?.address)`

**Review by katspaugh — COMMENTED** [review]
Nice!

Let's discuss it in the channel. It's a bit of a half-measure at this point. E.g. I would also add a "Confirm" button on each row.

**Comment by iamacook** [comment]
> Nice!
> 
> Let's discuss it in the channel. It's a bit of a half-measure at this point. E.g. I would also add a "Confirm" button on each row.

Sure. I'll convert this to a draft for the time being and revisit it later on.

File: src/components/dashboard/PendingTxs/PendingTxsList.tsx (hunk 1 above)
Comment (usame-algan) [inline src/components/dashboard/PendingTxs/PendingTxsList.tsx:71]: I think there is a util function called `isSignableBy`. Can it be reused here?

**Comment by katspaugh** [comment]
Let's close it for now, it can be restored if needed.

**Pull request closed without merging**